test/test_Cache.cpp

```diff
@@ -29,19 +31,20 @@ SC_MODULE(dut) {
   sc_signal<bool> pwrGood{"pwrGood", false};
   sc_signal<unsigned> framWaitStates{"framWaitStates", 1};
   tlm_utils::simple_initiator_socket<dut> cacheSocket{"cacheSocket"};
+  ClockSourceChannel clk{"clk", sc_time(1, SC_NS)};
```

Comment:
The cache is clocked from mclk only so maybe being explicit, i.e. clk -> mclk is better.

Reply:
that's only true for the msp target, the cache can be used in other targets too.